proto/report.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+
+// ReportApi provides methods to manage and retrieve reports. Reports are typically created by automations, such as
+// monthly metering reports, monthly fault reports etc. This API allows automations to provide these reports on demand.
+service ReportApi {
+  rpc ListReports(ListReportsRequest) returns (ListReportsResponse);
+  rpc DownloadReport(DownloadReportRequest) returns (stream ReportChunk);
```

Comment:
I'm not sure this download api will work in the browser, don't think streaming local file download is a thing. Other download apis we have use a GetDownloadURL rpc instead.

Reply:
Ah, ok. We should attempt to avoid any API that forces the browser to download and process the entire file at once, especially for this kind of thing. That's what the URL versions were made to do.